Sources/AccessibilitySnapshot/Core/Swift/Classes/SnapshotAndLegendView.swift

```diff
@@ -77,7 +77,7 @@ public class SnapshotAndLegendView: UIView {
                 y: Metrics.legendInsets.top
             )
 
-            let maxYBoundary = bounds.minY + availableLegendHeight
+            let maxYBoundary = bounds.minY + Metrics.legendInsets.top + availableLegendHeight
```

Comment:
This was an existing bug in the layout code for all accessibility snapshots, which could cause legend views to wrap to the next column too early.